unskript-ctl/unskript_ctl.py

```diff
@@ -604,12 +608,10 @@ def update_current_execution(status, id: str, content: dict):
         print("ERROR: Cannot Update Failed execution with No Content")
         return
 
-    failed_runbook = os.environ.get('EXECUTION_DIR').strip('"') + '/workspace/' + f"{id}.ipynb"
-
-
-    # If failed directory does not exists, lets create it
-    if os.path.exists(os.environ.get('EXECUTION_DIR').strip('"') + '/workspace') is False:
-        os.makedirs(os.environ.get('EXECUTION_DIR').strip('"') + '/workspace')
+    try:
```

Comment:
Why this try/except, it should just use the utils variable

Reply:
Having try/except isn't it much safer? anyway i have updated the anyway!